bin/git-forgit

```diff
@@ -38,6 +37,35 @@ $FZF_DEFAULT_OPTS
 $FORGIT_FZF_DEFAULT_OPTS
 "
 
+# If FORGIT_FZF_SHOW_NO_MATCHES is 1, then we don't pass `--exit-0` to fzf/fzf-tmux.
```

Comment:
Since this is really `forgit`'s behavior, we make it a forgit variable instead of making it implicit to the variables we pass to `fzf`.

Reply:
That's a good solution, thank you. IMO the variable name might be more self-explaining, though, and maybe we could invert the meaning. How do you like `FORGIT_EXIT_ON_EMPTY_LIST`, which would then default to 1?

Reply:
I think renaming the variable makes sense, and I did originally consider having it default to `1`, but when I thought about it, the convention amongst shell tools in general seems to be that having a variable completely unset should be implicitly falsy. I'm honestly still on the fence. I could go either way but I figured it might be worth making a comment and having at least a small back-and-forth about it.

Reply:
> I think renaming the variable makes sense, and I did originally consider having it default to 1, but when I thought about it, the convention amongst shell tools in general seems to be that having a variable completely unset should be implicitly falsy. I'm honestly still on the fence. I could go either way but I figured it might be worth making a comment and having at least a small back-and-forth about it.

@dljsjr That definitely makes sense and seems like a sane default behavior. So how about `FORGIT_SHOW_FZF_ON_EMPTY_LIST` then?